### ISSUE TITLE
GridContextMenu not working anymore

### ISSUE THREAD

**Issue by finaris-cs** [vaadin/flow-components#2821]
### Description

GridContextMenu is not shown anymore when defined for a grid. 
Worked well in older versions (can't say when it last worked, but somewhere between 14 and 21.x it did)

### Expected outcome

What did you expect to happen?

The defied menu for the grid should be shown (like it was working in older versions)

### Actual outcome

Default menu of the browser is opened instead

![image](https://user-images.githubusercontent.com/19190710/157194389-0e614c2b-9833-46fd-aabd-6a0455b09553.png)

### Minimal reproducible example

		Grid<String> grid = new Grid<>();

		grid.setHeight("300px");
		grid.setHeight("200px");

		grid.addColumn(s -> s).setFlexGrow(1);
		grid.setItems(List.of("Row 1", "Row 2", "Row 3"));

		// ((GridSingleSelectionModel<?>) grid.getSelectionModel()).setDeselectAllowed(false);

		GridContextMenu<String> menu = grid.addContextMenu();
		menu.addItem("XXX", System.out::println);

### Steps to reproduce

Use sample code, right click any grid row.

### Environment

- v22.0.7
- Windows 10

### Browsers Affected
- [x ] Chrome
- [ x] Firefox
- [ x] Edge
- not tested others

**Comment by web-padawan** [comment]
I could not reproduce this issue with latest Vaadin 22 branch using the code example provided: 

![grid-menu](https://user-images.githubusercontent.com/10589913/157205626-c35d02a5-77f8-46f4-90d5-6198c3238032.gif)

**Comment by finaris-cs** [comment]
Any idea what could cause this in my project?
Got menus on multiple grids, none of them is working anymore, neither dev nor production build.

Do i have any chance to debug JS in the browser to find the cause of this? If so, where to set the breakpoints?

**Comment by finaris-cs** [comment]
Extracted to a single page, still not working

`@Route(value = "test")
public class TestView extends Div {

	public TestView() {
		add(createGrid());
	}

	private Grid<String> createGrid() {
		Grid<String> grid = new Grid<>();

		grid.setHeight("300px");
		grid.setWidth("200px");

		grid.addColumn(s -> s).setFlexGrow(1);
		grid.setItems(List.of("Row 1", "Row 2", "Row 3"));

		// ((GridSingleSelectionModel<?>) grid.getSelectionModel()).setDeselectAllowed(false);

		GridContextMenu<String> menu = grid.addContextMenu();
		menu.addItem("XXX", System.out::println);
		return grid;
	}
}`

**Comment by sissbruecker** [comment]
Still can not reproduce this using the provided example code and Vaadin 22.0.8, Windows 10, Chrome 99. I attached a minimal project that I used for reproduction: [grid-context-menu-test.zip](https://github.com/vaadin/flow-components/files/8287605/grid-context-menu-test.zip)

If you want to debug this I would suggest to first clean your project, and then try removing things one by one to see what could interfere with the component. Or go the other way around, start from a clean slate and then add things until it breaks. Also check if disabling browser extensions makes a difference.
